crates/analyzer/src/handlers/check_expression.rs

```diff
@@ -307,9 +308,31 @@ impl VerylGrammarTrait for CheckExpression {
                         // TODO function check
                     }
                     IdentifierStatementGroup::Assignment(x) => {
+                        let token = arg.expression_identifier.identifier().token;
+                        let connect_operation = connect_operation_table::get(&token);
+
+                        let do_check = if connect_operation.is_none() {
+                            (true, true)
+                        } else if matches!(
+                            connect_operation.unwrap().rhs,
+                            ConnectOperand::Expression(_)
+                        ) {
+                            (true, false)
+                        } else {
+                            (false, false)
+                        };
+
+                        if !do_check.0 {
```

Comment:
Could add comment for the meaning of `do_check.0` and `do_check.1` or change them to struct?